stable/spinnaker/values.yaml

```diff
@@ -257,11 +257,14 @@ redis:
 minio:
   enabled: true
   image:
-    tag: RELEASE.2018-08-25T01-56-38Z
-  serviceType: ClusterIP
+    tag: RELEASE.2020-01-03T19-12-21Z
```

Comment:
i think the tag and service type configs have defaults in the upstream bitnami chart. Do we need these overridden?